CHANGES.rst

```diff
@@ -9,6 +9,8 @@ Change History for ZConfig
 
 - Run tests with pypy and pypy3 as well.
 
+- Host docs at http://zconfig.readthedocs.io
```

Comment:
Not for me!

![ekrano nuotrauka is 2017-01-20 17-23-44](https://cloud.githubusercontent.com/assets/159967/22154706/40185e10-df35-11e6-8be7-c9df369be98c.png)

Curious.